qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -212,37 +220,54 @@ rbd bench ${IMAGE} --io-type write --io-size=1024 --io-total=1024
 
 # unmap by image name test
 DEV=`_sudo rbd device --device-type nbd map ${POOL}/${IMAGE}`
-get_pid
+get_pid ${POOL}
 unmap_device ${IMAGE} ${PID}
 DEV=
 
 # map/unmap snap test
 rbd snap create ${POOL}/${IMAGE}@snap
 DEV=`_sudo rbd device --device-type nbd map ${POOL}/${IMAGE}@snap`
-get_pid
+get_pid ${POOL}
 unmap_device "${IMAGE}@snap" ${PID}
 DEV=
 
 # map/unmap namespace test
 rbd snap create ${POOL}/${NS}/${IMAGE}@snap
 DEV=`_sudo rbd device --device-type nbd map ${POOL}/${NS}/${IMAGE}@snap`
-get_pid ${NS}
+get_pid ${POOL} ${NS}
 unmap_device "${POOL}/${NS}/${IMAGE}@snap" ${PID}
 DEV=
 
 # unmap by image name test 2
 DEV=`_sudo rbd device --device-type nbd map ${POOL}/${IMAGE}`
-get_pid
+get_pid ${POOL}
 pid=$PID
 DEV=`_sudo rbd device --device-type nbd map ${POOL}/${NS}/${IMAGE}`
-get_pid ${NS}
+get_pid ${POOL} ${NS}
 unmap_device ${POOL}/${NS}/${IMAGE} ${PID}
 DEV=
 unmap_device ${POOL}/${IMAGE} ${pid}
 
+# map/unmap test with just image name
+DEV=`_sudo rbd device --device-type nbd map ${IMAGE}`
+get_pid ${POOL}
+unmap_device ${IMAGE} ${PID}
+DEV=
+
+#map/unmap test with just image name after changing default pool
+rbd config global set global rbd_default_pool ${ANOTHER_POOL}
+rbd create --size 10M ${IMAGE}
+DEV=`_sudo rbd device --device-type nbd map ${IMAGE}`
+get_pid ${ANOTHER_POOL}
+unmap_device ${IMAGE} ${PID}
+DEV=
+
+# reset
+rbd config global set global rbd_default_pool ${POOL}
```

Comment:
nit: it does not look like reset.

I would prefer `rbd config global rm global rbd_default_pool`. It will work correctly for all cases but one when that global was set to something before the test. But we probably may ignore it. If not, then a more complicate solution is to check the current setting using `rbd config global get global rbd_default_pool`, before changing and remember the result. And when resetting run either `rm` or `set ${OLD_DEFAULT_POOL}`.

Personally, I am fine with just `rbd config global rm global rbd_default_pool`

Reply:
Yeah, I think "rm" is sufficient.

Reply:
Done